packages/@aws-cdk/aws-events/lib/on-event-options.ts

```diff
@@ -39,4 +33,24 @@ export interface OnEventOptions {
    * https://docs.aws.amazon.com/eventbridge/latest/userguide/eventbridge-and-event-patterns.html
    */
   readonly eventPattern?: EventPattern;
+
+  /**
+   * The scope to use if the source of the rule and its target are in different Stacks.
+   * This helps dealing with cycles that often arise in these situations.
+   *
+   * @default - none (the main scope will be used, even for cross-stack Events)
+   */
+  readonly scopeIfCrossStack?: Construct;
```

Comment:
This name definitely sucks, so I'm very open to suggestions for a better one.

Reply:
Maybe just crossStackScope?

Reply:
OK, I'll update to `crossStackScope`.